descqa/AbsMagCorrelation.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import print_function, division, unicode_literals, absolute_import
```

Comment:
This file is almost identical to `descqa/AngularCorrelation.py` --- we should have only one of them and create some test options to accommodate the difference